src/System.Runtime/tests/System/DateTimeTests.cs

```diff
@@ -987,9 +987,8 @@ public static IEnumerable<object[]> Parse_ValidInput_Suceeds_MemberData()
                 yield return new object[] { today.ToString(hebrewCulture), hebrewCulture, today };
             }
 
-            // [ActiveIssue(23922)] // This is failing on Windows 8.1
-            //var mongolianCulture = new CultureInfo("mn-MN");
-            //yield return new object[] { today.ToString(mongolianCulture.DateTimeFormat.FullDateTimePattern, mongolianCulture), mongolianCulture, today };
+            var mongolianCulture = new CultureInfo("mn-MN");
+            yield return new object[] { today.ToString(mongolianCulture), mongolianCulture, today };
```

Comment:
FYI, one of the reasons I used FullDateTimePattern was that it exercised code paths not being tested anywhere else.

Reply:
Interesting. Is the hit code path was because of using Mongolian? I am asking because I am seeing we have other tests using FullDateTimePattern.

https://github.com/dotnet/corefx/blob/dd62712ca69f0c817f6ae59eb30236bc2eaec795/src/System.Runtime/tests/System/DateTimeTests.cs#L838

Reply:
Yes, or at least I believe that's what I saw and why I chose mn-MN.  It's possible I was just wrong, of course :)